packages/docs/celo-codebase/protocol/identity/meta-transaction-wallet.md

```diff
@@ -0,0 +1,23 @@
+# Meta-Transaction Wallet
```

Comment:
Is the relevant piece for folks the MTW or the fact that there are "account address" vs. "walletAddress"

Reply:
Renaming to "Valora Accounts" to be more clear to readers.